packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -88,17 +104,26 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
   </div>: null
   
   useEffect(() => {
-    if (adminBranch == -1 && currentUser?.isAdmin) {
+    if (adminBranch === -1 && currentUser?.isAdmin) {
       setAdminBranch(randInt(5));
     }
   }, [adminBranch, currentUser?.isAdmin]);
-  
-  if (forumTypeSetting.get() === 'EAForum') return null
-  
+
+  // disable on AlignmentForum
+  if (forumTypeSetting.get() === 'AlignmentForum') {
+    return null;
+  }
+
   // Placeholder to prevent SSR mismatch, changed on load.
-  if (adminBranch == -1)
+  if (adminBranch === -1 && currentUser?.isAdmin)
     return <div/>
```

Comment:
This was the main issue, which prevented the component from displaying for non-admins

Reply:
Oh goddammit. Thank you very much for catching this (the bug was written by me).